src/pathwayvis/components/map/views/map.component.html

```diff
@@ -1,7 +1,7 @@
 <md-whiteframe class="map-context-menu md-whiteframe-4dp" layout="column" layout-align="center center" ng-cloak style="visibility: hidden">
   <md-menu-content class="context-menu">
     <md-menu-item ng-repeat="action in ctrl.contextActions">
-      <md-button ng-click="ctrl.processActionClick(action, ctrl.contextElement)">{{action.label}}</md-button>
+      <md-button ng-click="ctrl.processActionClick(action, ctrl.contextElement)">{{JSON.stringfy(ctrl.contextActions.shared)}}</md-button>
```

Comment:
It'd be nice not to add logic to templates. Could you move this to the controller?